packages/core/src/overlay/OverlayHeader.tsx

```diff
@@ -0,0 +1,84 @@
+import { useComponentCssInjection } from "@salt-ds/styles";
+import { useWindow } from "@salt-ds/window";
+import clsx from "clsx";
+import {
+  type ComponentPropsWithoutRef,
+  type ReactNode,
+  forwardRef,
+} from "react";
+import { H3, Text } from "../text";
+import { makePrefixer } from "../utils";
+import overlayHeaderCss from "./OverlayHeader.css";
+
+const withBaseName = makePrefixer("saltOverlayHeader");
+
+export interface OverlayPanelContentProps
+  extends ComponentPropsWithoutRef<"div"> {
+  /**
+   * Description text is displayed just below the header
+   **/
+  description?: string;
+  /**
+   * End adornment component
+   */
+  endAdornment?: ReactNode;
```

Comment:
Using `endAdornment` to be consistent with other components. Are there any strong opinions about the name? @joshwooding @origami-z

Reply:
adornment is really only form controls previously... how does design refer to this? @navkaur76

Reply:
Adornment is also used in grid corner triangle? @bhoppers2008